recipes/nagl/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "nagl" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimonBoothroyd/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cca4b5c975405d720368a337f2908586ef44185934f7ac56c98bb4d078151153
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - openff-utilities
+    - pydantic
+    - click
+    - click-option-group
+    - pytorch
+    - tqdm
+
+test:
+  imports:
+    - nagl
+
+about:
+  home: https://github.com/SimonBoothroyd/nagl
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Probably should include the 3rd party stuff too:
```suggestion
  license: MIT AND BSD-3-CLAUSE
  license_file:
    - LICENSE
    - LICENSE-3RD-PARTY
```